scripts/get-cdnjs-list.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+import json
+import subprocess
+
+links = {}
+data = subprocess.check_output(['links', '-dump', 'https://cdnjs.com/'])
```

Comment:
We do not have to scrape the website.  There is a nice .json file we could use with a little parsing (might need an input format change to account for different files and versions): https://github.com/cdnjs/cdnjs/wiki/Extensions%2C-Plugins%2C-Resources#developers